src/components/EventListCard/Modal/EventListCardModals.spec.tsx

```
@@ -63,7 +63,7 @@ vi.mock('utils/errorHandler', () => ({
 vi.mock('./Preview/EventListCardPreviewModal', () => ({
   default: vi.fn(),
 }));
-vi.mock('./Delete/EventListCardDeleteModal', () => ({
+vi.mock('shared-components/EventListCard/Modal/Delete/EventListCardDeleteModal', () => ({
```

ESLint

[error] Replace `'shared-components/EventListCard/Modal/Delete/EventListCardDeleteModal',` with `⏎··'shared-components/EventListCard/Modal/Delete/EventListCardDeleteModal',⏎·`

(prettier/prettier)